src/components/Fonts.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react"
+import { Global } from "@emotion/react"
+
+const Fonts = () => (
+  <Global
+    styles={`
+      /* latin */
+      @font-face {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 300 800;
+        font-display: swap;
+        src: url('/fonts/Inter.ttf') format('truetype');
+      }
+      @font-face {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 300 800;
+        font-display: swap;
+        font-named-instance: 'Regular';
+        src: url("/fonts/Inter-roman.var.woff2") format("woff2");
+      }
+      `}
+  />
+)
```

Comment:
Sounds good. I can update this!

Reply:
If we are planning on using them in the future then lets add it, yes.

The browser will only load those fonts that are used on the page. If no element uses that font, it will not load it. We should be fine on that aspect.